tokens/src/themes/light/components/Annotation.json

```diff
@@ -0,0 +1,28 @@
+{
+  "elevation": {
+    "annotation": {
+      "box-shadow": {
```

Comment:
It looks like the `type` attribute in these tokens might have gotten lost somehow (in the original, now-deleted Annotation.json, the `type` attributes are there. I wonder if it _may_ be related to the issues we ran into trying to get `alpha` brought in sync with `master` that caused us to essentially rewrite the git history for the `alpha` branch... It may be worth a check to see if there might be anything else missing in addition to the `type`? It does look like this may be an issue in many of the files.